boards/riscv/esp32c3_devkitm/doc/index.rst

```diff
@@ -35,104 +35,88 @@ The features include the following:
 System requirements
 *******************
 
-Build Environment Setup
-=======================
+Prerequisites
+-------------
 
-Some variables must be exported into the environment prior to building this port.
-Find more information at :ref:`env_vars` on how to keep this settings saved in you environment.
+Espressif HAL requires binary blobs in order work. The west extension below performs the required
+syncronization to clone, checkout and pull the submodules:
+
+.. code-block:: console
+
+   west espressif update
 
 .. note::
 
-   In case of manual toolchain installation, set :file:`ESPRESSIF_TOOLCHAIN_PATH` accordingly.
-   Otherwise, set toolchain path as below. If necessary.
+    It is recommended running the command above after :file:`west update`.
 
-On Linux and macOS:
+Building & Flashing
+-------------------
 
-.. code-block:: console
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: esp32c3_devkitm
+   :goals: build
 
-   export ZEPHYR_TOOLCHAIN_VARIANT="espressif"
-   export ESPRESSIF_TOOLCHAIN_PATH="${HOME}/.espressif/tools/zephyr"
+The usual ``flash`` target will work with the ``esp32c3_devkitm`` board
+configuration. Here is an example for the :ref:`hello_world`
+application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: esp32c3_devkitm
+   :goals: flash
 
-On Windows:
+Open the serial monitor using the following command:
+
+.. code-block:: shell
+
+   west espressif monitor
+
+After the board has automatically reset and booted, you should see the following
+message in the monitor:
 
 .. code-block:: console
 
-  # on CMD:
-  set ESPRESSIF_TOOLCHAIN_PATH=%USERPROFILE%\.espressif\tools\zephyr
-  set ZEPHYR_TOOLCHAIN_VARIANT=espressif
+   ***** Booting Zephyr OS vx.x.x-xxx-gxxxxxxxxxxxx *****
+   Hello World! esp32c3_devkitm
 
-  # on PowerShell
-  $env:ESPRESSIF_TOOLCHAIN_PATH="$env:USERPROFILE\.espressif\tools\zephyr"
-  $env:ZEPHYR_TOOLCHAIN_VARIANT="espressif"
+Debugging
+---------
 
-Finally, retrieve required submodules to build this port. This might take a while for the first time:
+As with much custom hardware, the ESP32C3 modules require patches to
+OpenOCD that are not upstreamed. Espressif maintains their own fork of
+the project. The custom OpenOCD can be obtained by running the following extension:
 
 .. code-block:: console
 
-   west espressif update
+   west espressif install
 
-.. note::
+   .. note::
```

Comment:
This is not correct. Need to remove leading spaces.

Reply:
Can this be added into documentation's CI check?

Reply:
It is already clear if you look at the PR doc build:
https://builds.zephyrproject.io/zephyr/pr/43534/docs/boards/riscv/esp32c3_devkitm/doc/index.html